Shared/Settings.swift

```diff
@@ -54,4 +54,13 @@ struct Settings {
         prefs.set(value, forKey: toggle.rawValue)
         prefs.synchronize()
     }
+    
+    static func siriRequestsErase() -> Bool {
+        return prefs.bool(forKey: "siriRequestsErase")
```

Comment:
Should these keys be declared as constants at the top of this or perhaps in UIConstants?